code/game/objects/structures/window.dm

```diff
@@ -231,6 +234,7 @@
 		"Smashed against \the [src] by [key_name(user)]",
 		"smashed [key_name(target)] against \the [src]."
 	)
+	sleep(5) //Allow a littleanimating time
```

Comment:
Without this sleep, the attack animation doesn't work because it gets immediately overridden by the grab-dropping animation